src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -2414,6 +2414,101 @@ public GCCondemnedReasons[] PerHeapCondemnedReasons
                 return _PerHeapCondemnedReasons;
             }
         }
+
+        public enum TimingType
+        {
+            /// <summary>
+            /// This field records the time spent for marking a type of GCHandle named HNDTYPE_SIZEDREF.
+            /// This happens only for blocking GC, and for majority of the time, this does not happen (because there are no sizedref handles)
+            /// so you get a 0 for blocking GC and a Null for background GC.
+            ///
+            /// Available for .NET 6+
+            /// </summary>
+            MarkSizedRef,
```

Comment:
since this is not in use, we could just leave it out. it's fine to have the code in GC but in tooling it's probably better to leave it out since you'll never get a non zero value.